### DISCUSSION TITLE
Support for Industrial SoM

### DISCUSSION THREAD

**Discussion by ewhac** [OE4T/meta-tegra#990] [Q&A]
We have a customized carrier board for which we have all the appropriate support set up to build a tegraflash-style image with NVIDIA's L4T 32.6.x suite.  However, we wish to migrate to Yocto to more closely control the distro.

We've had success porting our config for the commercial SOM, but meta-tegra does not yet have turnkey recipes or examples for the industrial SOM.  A quick diff in L4T 32.6.1 between `jetson-agx-xavier-devkit.conf` and `jetson-agx-xavier-industrial.conf` suggests there's more involved than simply copying additional environment variables into a `flashvars` file.

Is anyone already working on this?  Can anyone offer any suggestions/guidance on how best to approach the port?

**Comment by plyons72** [comment]
I'm working on this now. No solutions just yet. I've updated `flashvars`,`tegra194-flash-helper.sh`, and created `conf/machine/jetson-agx-xavier-industrial.conf`. `doflash.sh` script seems to run quickly... 40ish seconds down from 600ish seconds... and the board is not resetting at the end of it. I'm working through this issue as I think more changes need to be made on my end. Did you ever find a solution?
